Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/AgendaDayFragment.java

```diff
@@ -131,9 +177,7 @@ private void setupHeaderAdapter() {
                                     return false;
                                 }
                                 Bundle arguments = new Bundle();
-                                arguments.putString(ScheduleDatabase.NAME, item.getItemData().speakerName);
-                                arguments.putString(ScheduleDatabase.TALK_TIME, item.getItemData().time);
-                                arguments.putString(ScheduleDatabase.ROOM, item.getItemData().room);
+                                arguments.putString(ScheduleDatabase.SCHEDULE_ITEM_ROW, gson.toJson(itemData, ScheduleRow.class));
```

Comment:
could probably pass the event ID through instead of the entire row...firebase lookups are supposedly fast?

Reply:
Yea, but distinct firebase lookup for data we already have available? Firebase would also have to redo the serialization as well - don't think this is faster considering we already have the data we need?